lib/tests/test.js

```diff
@@ -200,10 +201,17 @@ class Test {
         return cb();
       }
       this.options.node = node;
+      this.showNodeHttpWarning();
       cb();
     });
   }
 
+  showNodeHttpWarning() {
+    if (this.options.node.startsWith('http')) {
+      this.engine.logger.warn("You are using http to connect to the node, as a result the gas details won't be correct");
```

Comment:
Could we maybe add a possible solution for the user? For example, 
> You are using http to connect to the node, as a result the gas details won't be correct. For correct gas details reporting, please use a websockets connection to your node.

What do you think of this?

Reply:
Much better indeed, I updated the text